src/Homie/Boot/BootConfig.cpp

```diff
@@ -143,8 +144,8 @@ void BootConfig::_onWifiConnectRequest(AsyncWebServerRequest *request) {
 void BootConfig::_onWifiStatusRequest(AsyncWebServerRequest *request) {
   Interface::get().getLogger() << F("Received Wi-Fi status request") << endl;
 
-  DynamicJsonBuffer generatedJsonBuffer(JSON_OBJECT_SIZE(2));
-  JsonObject& json = generatedJsonBuffer.createObject();
+  StaticJsonDocument<JSON_OBJECT_SIZE(2) + 18 + 16> generatedJsonDoc; // Includes memory to duplicate strings for status and local_ip
```

Comment:
Please explain how you calculated these numbers.

Are only IPv4 addresses allowed? If not we run in trouble with IPv6 addresses.

You might also place your comment above the line. This increases the readability.

Reply:
Currently, `WiFi.localIP()` only returns an IPv4 address. But I can change the value to 40 to fit IPv6 addresses in the future.